src/dcs.rs

```diff
@@ -0,0 +1,106 @@
+use display_interface::{DataFormat, WriteOnlyDataCommand};
+
+use crate::{instruction::Instruction, Error, TearingEffect};
+
+#[macro_use]
+mod macros;
+
+mod madctl;
+pub use madctl::*;
+mod colmod;
+pub use colmod::*;
+mod caset;
+pub use caset::*;
+mod raset;
+pub use raset::*;
+mod vscrdef;
+pub use vscrdef::*;
+mod vscad;
+pub use vscad::*;
+
+///
+/// Provides a constructor for complex commands
+/// e.g. `Madctl::new().with_bgr(true).bytes()`
+///
```

Comment:
The trait is implemented for all commands, not only the complex ones.

Reply:
fixed